css/newTabPage.css

```diff
@@ -17,6 +17,12 @@ body:not(.is-ntp) #ntp-content {
     height: 100%;
     margin: auto;
     object-fit: cover;
+    transition: 0.2s filter, 0.2s transform;
+}
+
+.ntp-blur #ntp-background {
+    filter: blur(12px);
+    transform: scale(1.1);
```

Comment:
This makes sense to eliminate the white edges around the blur, but I think it's pretty distracting. What if we:
* Reduce the magnitude to `scale(1.03)` - it looks like this is still enough to eliminate the white edges
* Apply the scale to the image all the time, so that there's no motion when the blur is applied?